run-checks

```diff
@@ -365,6 +365,20 @@ if [ -n "$SPREAD" ]; then
             exit 1
     esac
 
+    if ./check-pr-has-label.py "$TRAVIS_PULL_REQUEST" "LABEL_RUN_SPREAD_NESTED"; then
+        case "$SPREAD" in
+            full)
+                spread "google-nested:tests/nested/"
+            ;;
+            ubuntu-only)
+                spread "google-nested:[u]...:tests/nested/"
```

Comment:
What do the dots match?

Reply:
I'm wondering the same

Reply:
iirc it is used to expand in golang

Reply:
I'm assuming this matching is not RE, and expands to all systems starting with "u" (i.e. all ubuntus), is that correct?

Reply:
So I looked at the spread code, three dots is one of a few special expressions that are translated into a regex; in this case it translates to a regex matching all chars until ":". So yes, it means all ubuntus.

Reply:
I checked there as well, it seems that the brackets around "u" are unnecessary in this case.